src/fireedge/src/modules/features/General/actions.js

```diff
@@ -0,0 +1,54 @@
+/* ------------------------------------------------------------------------- *
+ * Copyright 2002-2025, OpenNebula Project, OpenNebula Systems               *
+ *                                                                           *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may   *
+ * not use this file except in compliance with the License. You may obtain   *
+ * a copy of the License at                                                  *
+ *                                                                           *
+ * http://www.apache.org/licenses/LICENSE-2.0                                *
+ *                                                                           *
+ * Unless required by applicable law or agreed to in writing, software       *
+ * distributed under the License is distributed on an "AS IS" BASIS,         *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  *
+ * See the License for the specific language governing permissions and       *
+ * limitations under the License.                                            *
+ * ------------------------------------------------------------------------- */
+import { createAction } from '@reduxjs/toolkit'
+
+export const fixMenu = createAction('Fix menu')
+export const changeZone = createAction('Change zone')
+export const changeLoading = createAction('Change loading')
+export const changeAppTitle = createAction('Change App title')
+export const setSelectedIds = createAction('Set selected IDs')
+export const setUpdateDialog = createAction('Set update dialog')
+export const setFullMode = createAction('Set full mode')
+export const setBreadcrumb = createAction('Set breadcrumb')
+export const setTableViewMode = createAction('Set table view mode')
+
+export const updateDisabledSteps = createAction('Set disabled steps')
+export const setLoadOsProfile = createAction('Set load OS profile')
+export const resetLoadOsProfile = createAction('Reset load OS profiles')
+export const dismissSnackbar = createAction('Dismiss snackbar')
+export const deleteSnackbar = createAction('Delete snackbar')
+export const setUploadSnackbar = createAction('Change upload snackbar')
+export const setFieldPath = createAction('Set dynamic field path')
+export const resetFieldPath = createAction('Reset field path')
+export const initModifiedFields = createAction('Init modified fields')
+export const changePositionModifiedFields = createAction(
+  'Change position of two array elements in modified fields'
+)
+export const setModifiedFields = createAction(
+  'Set modified fields',
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This route is missing authorization checks. Without proper authorization, sensitive routes may be exposed to unauthorized users, leading to potential data breaches or unauthorized actions. Ensure that authorization middleware or handlers are applied. For more information on missing authorization vulnerabilities, refer to [CWE-862](https://cwe.mitre.org/data/definitions/862.html).